src/test/java/com/wikia/webdriver/common/driverprovider/mobileprovider/MobileDriverProvider.java

```diff
@@ -30,15 +30,11 @@ public WebDriver getDriverInstance() {
         driver = getChromeDriver();
         break;
       case "IOS":
-        //@TODO
+        // @TODO
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)